Patch-Source: https://gitlab.gnome.org/World/Phosh/gmobile/-/merge_requests/72
---
From 4c9e7aee8a8050fd22882658b0528e6f6cbb0a95 Mon Sep 17 00:00:00 2001
From: Evangelos Ribeiro Tzaras <devrtz@fortysixandtwo.eu>
Date: Thu, 8 May 2025 16:55:30 +0200
Subject: [PATCH] data/hwdb: Insert empty line to indicate a new input device

Otherwise it will get ignored (slightly edited for readability):

$ sudo systemd-hwdb update
/usr/lib/udev/hwdb.d/61-gmobile-wakeup.hwdb:110:
  Property or empty line expected, got "gmobile:name:PinePhone 1c22e00.codec-jack:dt:pine64,pinephone-1.*", ignoring record.
/usr/lib/udev/hwdb.d/61-gmobile-wakeup.hwdb:111:
  Match expected but got indented property " GM_WAKEUP_KEY_164=0", ignoring line.

Fixes b1cd28b89a105b83b3b485c28f94af0b53dc6966

Part-of: <https://gitlab.gnome.org/World/Phosh/gmobile/-/merge_requests/72>
---
 data/61-gmobile-wakeup.hwdb | 1 +
 1 file changed, 1 insertion(+)

diff --git a/data/61-gmobile-wakeup.hwdb b/data/61-gmobile-wakeup.hwdb
index e056c65..de09ac2 100644
--- a/data/61-gmobile-wakeup.hwdb
+++ b/data/61-gmobile-wakeup.hwdb
@@ -103,6 +103,7 @@ gmobile:name:Volume keys:dt:oneplus,fajita*
 # PinePhone
 gmobile:name:1c21800.lradc:dt:pine64,pinephone-1.*
  GM_WAKEUP_KEY_DEFAULT=0
+
 gmobile:name:PinePhone 1c22e00.codec-jack:dt:pine64,pinephone-1.*
  GM_WAKEUP_KEY_164=0
 
-- 
GitLab

